docs/self-managed/operational-guides/update-guide/860-to-870.md

```diff
@@ -59,3 +59,84 @@ In that case, we append the following records:
 - `ASSIGN`
 - `ASSIGNING`
 - `ASSIGNED`
+
+## Prefix migration
+
+If a custom prefix has been defined for Operate and Tasklist (for example, using `camunda.operate.elasticsearch.indexPrefix`), then data must be migrated manually to the new indices.
+
+### Make the Elasticsearch instance reachable
+
+The migration relies on communicating directly with Elasticsearch. The Elasticsearch instance must be reachable on `http://localhost:9200`.
+
+- **Kubernetes**: Port-forward your Elasticsearch instance to ensure it is reachable. In the following example, `camunda-platform` can be replaced by the release name:
+
+  ```shell
+  kubectl port-forward svc/camunda-platform-elasticsearch 9200:9200
+  ```
+
+### Create harmonized indices
+
+To begin creating the harmonized indices for migration, first pull the following Docker image:
+
+```shell
+docker pull camunda/zeebe:SNAPSHOT
+```
+
+Run the command below, replacing `CAMUNDA_DATABASE_PREFIX` value with your desired prefix:
+
+```shell
+docker run --rm -it --env CAMUNDA_DATABASE_URL="host.docker.internal:9200" --env CAMUNDA_DATABASE_INDEXPREFIX=<prefix> --entrypoint /usr/local/zeebe/bin/schema camunda/zeebe:SNAPSHOT
+```
```

Comment:
❓ Why is this necessary?

Reply:
Yeah I just later realized that we need to do this, because of the reindex. Could we mention this?